plugins/statistics.koplugin/main.lua

```diff
@@ -172,6 +195,16 @@ function ReaderStatistics:addToMainMenu(tab_item_table)
                     })
                 end
             },
+            {
+                text = _("Reading Statistics"),
```

Comment:
I think we need a more descriptive name here. Perhaps `Time range` or `Statistics by time`?